examples/cython/cython_main.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import ray
+import click
+import inspect
+
+import numpy as np
+import cython_examples as cyth
+
+
+def run_func(func, *args, **kwargs):
+
+    ray.init()
+
+    func = ray.remote(func)
+
+    # NOTE: kwargs not allowed for now
+    result = ray.get(func.remote(*args))
+
+    # Inspect the stack to get calling example
+    caller = inspect.stack()[1][3]
+    print('%s: %s' % (caller, str(result)))
+
+    return result
+
+
+@click.group(context_settings=dict(help_option_names=['-h', '--help']))
+def cli():
+    """
+    Working with Cython actors and functions in Ray
+    """
+
+
+@cli.command()
+def example1():
+    "Cython def function"
```

Comment:
Is there a particular reason for 1 quote commenting (as opposed to three from above?)

Reply:
Good catch! I also noticed that ray prefers double to single quotes where possible, so I changed all those as well.